jest.unit.config.ts

```diff
@@ -15,8 +15,8 @@ const config: Config.InitialOptions = {
       statements: 60
     }
   }*/
-  globalSetup: './test/jest.setup.ts',
-  globalTeardown: './test/jest.teardown.ts',
+  globalSetup: './test/jest.unit.setup.ts',
+  globalTeardown: './test/jest.unit.teardown.ts',
```

Comment:
I renamed jest.config.ts to jest.unit.config.ts otherwise it kind of seemed like this was the master jest config files